source/index.blade.php

```diff
@@ -545,7 +549,7 @@ class="ud-contact-form-wrapper wow fadeInUp"
     <!-- ====== Footer End ====== -->
 
     <!-- ====== Back To Top Start ====== -->
-    <a href="javascript:void(0)" class="back-to-top">
+    <a href="javascript:void(0)" class="back-to-top" onclick="backToTop()">
```

Comment:
Look the comment below.

```suggestion
    <a id="back-to-top" class="back-to-top">
```